application/language/spanish/qso_lang.php

```diff
@@ -26,74 +26,74 @@
+$lang['qso_simplefle_operator'] = "Operador";
+$lang['qso_simplefle_operator_hint'] = "ej. OK2CQR";
+$lang['qso_simplefle_station_call_location'] = "Indicativo/Localización de la Estación";
+$lang['qso_simplefle_station_call_location_hint'] = "Si operó desde una nueva localización, primero cree una nueva <a href=". site_url('station') . ">Localización de Estación</a>";
+$lang['qso_simplefle_utc_time'] = "Hora UTC Actual";
+$lang['qso_simplefle_enter_the_data'] = "Ingrese los Datos";
+$lang['qso_simplefle_syntax_help_close_w_sample'] = "Cerrar y Cargar Datos de Muestra";
+$lang['qso_simplefle_reload'] = "Recargar Lista de QSO";
+$lang['qso_simplefle_save'] = "Guardar en Cloudlog";
+$lang['qso_simplefle_clear'] = "Limpiar Sesión de Registro";
+$lang['qso_simplefle_refs_hint'] = "Las Referencias deben ser <u>S</u>OTA, <u>I</u>OTA, <u>P</u>OTA o <u>W</u>WFF";
+
+$lang['qso_simplefle_error_band'] = "¡Falta la Banda!";
+$lang['qso_simplefle_error_mode'] = "¡Falta el Modo!";
+$lang['qso_simplefle_error_time'] = "¡No se configuró la Hora!";
+$lang['qso_simplefle_error_stationcall'] = "El Indicativo de la Estación no fue seleccionado";
+$lang['qso_simplefle_error_operator'] = "El campo 'Operador' está vacío";
+$lang['qso_simplefle_warning_reset'] = "¡Advertencia! ¿Está seguro que desea reinicializar todo?";
+$lang['qso_simplefle_warning_missing_band_mode'] = "¡Advertencia! ¡No puede registrar la lista de QSO, porque algunos QSO no tienen banda y/o modo definido!";
+$lang['qso_simplefle_warning_missing_time'] = "¡Advertencia! ¡No puede registrar la lista de QSO, porque algunos QSO no tienen hora definida!";
+$lang['qso_simplefle_warning_example_data'] = "¡Atención! ¡El campo Datos contiene datos de muestra. Primero Limpie la Sesión de Registro!";
+$lang['qso_simplefle_confirm_save_to_log'] = "¿Está seguro que desea adicionar estos QSO al Libro de Guardia y limpiar la sesión?";
+$lang['qso_simplefle_success_save_to_log_header'] = "¡QSOs Registradas!";
+$lang['qso_simplefle_success_save_to_log'] = "¡Las QSO fueron satisfactoriamente guardadas en el Libro de Guardia!";
+$lang['qso_simplefle_error_date'] = "Fecha inválida";
+
+$lang['qso_simplefle_syntax_help_button'] = "Ayuda de Sintaxis";
+$lang['qso_simplefle_syntax_help_title'] = "Sintaxis para FLE";
+$lang['qso_simplefle_syntax_help_ln1'] = "Antes de iniciar a registrar un QSO, note las reglas básicas.";
+$lang['qso_simplefle_syntax_help_ln2'] = "- Cada QSO nuevo debe estar en una línea nueva.";
+$lang['qso_simplefle_syntax_help_ln3'] = "- En cada nueva línea, solo escriba datos que hayan cambiado desde el QSO anterior.";
+$lang['qso_simplefle_syntax_help_ln4'] = "Para comenzar, asegúrese que ya ha llenado el formulario en la izquierda con la fecha, el indicativo de la estación y el indicativo del operador. Los datos principales incluyen la banda (o QRG en MHz, ej., '7.145'), modo, y hora. Después de la hora, usted debe proveer el primer QSO, que es esencialmente el indicativo.";
```

Comment:
```suggestion
$lang['qso_simplefle_syntax_help_ln4'] = "Para comenzar, asegúrese que ya ha rellenado el formulario en la izquierda con la fecha, el indicativo de la estación y el indicativo del operador. Los datos principales incluyen la banda (o QRG en MHz, ej., '7.145'), modo, y hora. Después de la hora, usted debe proveer el primer QSO, que es esencialmente el indicativo.";
```

And probably there is something better than "proveer", such as "introducir".